src/ucp/rndv/proto_rndv.c

```diff
@@ -403,6 +403,7 @@ static void ucp_proto_rndv_ctrl_variant_probe(
         cfg_thresh = remote_proto->cfg_thresh;
     }
 
+    ucp_proto_perf_apply_bias(perf, params->perf_bias);
```

Comment:
don't we need to apply it for RTS only? E.g. now it is applied for RTR as well making it more preferable than get_zcopy

Reply:
I thought it is done in this way according to this comment: https://github.com/openucx/ucx/pull/10292#issuecomment-2469830879.

`ucp_proto_rndv_ctrl_init_params_t::perf_bias` is controlled by `UCX_RNDV_PERF_DIFF` in RTS protocol, and `ucp_proto_rndv_ctrl_init_params_t::perf_bias` is set to zero in RTR protocol.